packages/storybook8/stories/Concepts/TeamsInterop/JoinMeetingWithLink/JoinMeetingWithLink.stories.tsx

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Meta } from '@storybook/react';
+import React, { useState, useEffect } from 'react';
+import { Stack } from '@fluentui/react';
+import { CallWithChatExampleProps } from './snippets/CallWithChat.snippet';
+import { CallWithChatExperienceWithErrorChecks } from './snippets/CallWithChatWithErrorChecks.snippet';
+import { ConfigJoinMeetingHintBanner } from '../utils/Utils';
+import { compositeExperienceContainerStyle } from '../../constants';
+import { controlsToAdd, ArgsFrom, defaultCallWithChatCompositeHiddenControls } from '../utils/controlsUtils';
+
+const storyControls = {
+  userId: controlsToAdd.userId,
+  token: controlsToAdd.token,
+  endpointUrl: controlsToAdd.endpointUrl,
+  displayName: controlsToAdd.requiredDisplayName,
+  teamsMeetingLink: controlsToAdd.teamsMeetingLink,
+  formFactor: controlsToAdd.formFactor
+};
+
+const JoinWithMeetingLinkStory = (args: ArgsFrom<typeof storyControls>, context): JSX.Element => {
+  const [meetingProps, setMeetingProps] = useState<CallWithChatExampleProps>();
+
+  useEffect(() => {
+    const fetchToken = async (): Promise<void> => {
+      if (!!args.token && !!args.userId && !!args.endpointUrl && !!args.displayName && !!args.teamsMeetingLink) {
+        setMeetingProps({
+          userId: { communicationUserId: args.userId },
+          token: args.token,
+          displayName: args.displayName,
+          endpointUrl: args.endpointUrl,
+          locator: { meetingLink: args.teamsMeetingLink }
+        });
+      } else {
+        setMeetingProps(undefined);
+      }
+    };
+    fetchToken();
+  }, [args.token, args.userId, args.endpointUrl, args.displayName, args.teamsMeetingLink]);
+
+  return (
+    <>
+      <Stack horizontalAlign="center" verticalAlign="center" styles={compositeExperienceContainerStyle}>
+        {meetingProps ? (
+          <CallWithChatExperienceWithErrorChecks
+            fluentTheme={context.theme}
+            rtl={context.globals.rtl === 'rtl'}
+            {...meetingProps}
+          />
+        ) : (
+          <ConfigJoinMeetingHintBanner />
+        )}
+      </Stack>
+    </>
+  );
+};
+
+export const JoinMeetingWithLink = JoinWithMeetingLinkStory.bind({});
+
+export default {
+  title: 'Concepts/Teams Interop/Join Meeting With Link',
+  component: JoinMeetingWithLink,
+  argTypes: {
+    ...storyControls,
+    ...defaultCallWithChatCompositeHiddenControls
+  }
```

Comment:
need to set default values using `args: {}`
<img width="776" alt="Screenshot 2024-07-29 at 3 43 30 PM" src="https://github.com/user-attachments/assets/6770e833-bca9-443c-a6f7-27499733c6b2">

Reply:
will look into this

Reply:
added it in now